torch/nn/attention/flex_attention.py

```diff
@@ -1030,11 +1144,13 @@ def score_mod(
                 f"but got Hq={Hq} and Hkv={Hkv}."
             )
 
+    from torch.fx.experimental.symbolic_shapes import is_nested_int
+
     if score_mod is None:
         score_mod = _identity
     if block_mask is None:
         block_mask = _create_empty_block_mask(query, key)
-    elif (
+    elif not is_nested_int(query.size(-2)) and (
```

Comment:
I hacked around this but is this important to have for NJT as well ? The issue is that the logic at this level is before the HOP is called, and the HOP is where the NJT is unwrapped to pass the `values` buffer to flex. So `query` is still an NJT at this point, `query.size(-2)` is usually a nested int, and nested ints don't support the lt operator.